[INFO]2021-03-03 15:09:34,263: log files at D:\python_lx\akm\common\log
[INFO]2021-03-03 15:09:34,263: find module agvCtrl.main_agv
[INFO]2021-03-03 15:09:34,263: find module agvDevice.main_agvDevice
[INFO]2021-03-03 15:09:34,264: find module alarm.main_alarm
[INFO]2021-03-03 15:09:34,264: find module auth.main_auth
[INFO]2021-03-03 15:09:34,264: find module button.main_button
[INFO]2021-03-03 15:09:34,264: find module door.main_door
[INFO]2021-03-03 15:09:34,264: find module elevator.main_elevator
[INFO]2021-03-03 15:09:34,265: find module virtualDoor.main_doorCtrl
[INFO]2021-03-03 15:09:34,265: import agvCtrl.main_agv
[INFO]2021-03-03 15:09:34,631: create lock <lock name:doorMgr.lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C32BD630>>
[INFO]2021-03-03 15:09:34,637: create lock <lock name:door.lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C32CD6F0>>
[INFO]2021-03-03 15:09:34,637: create lock <lock name:door.lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C32CDB70>>
[INFO]2021-03-03 15:09:34,734: create lock <lock name:timer.lock_2580258307808,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C3624060>>
[ERROR]2021-03-03 15:09:34,743: mqtt is disabled
[INFO]2021-03-03 15:09:34,744: mqtt: subscribe [alarm/raise/+]
[INFO]2021-03-03 15:09:34,744: mqtt: subscribe [alarm/clear/+]
[INFO]2021-03-03 15:09:37,777: Mongodb connect to 127.0.0.1 port 27017
[INFO]2021-03-03 15:09:37,832: create lock <lock name:elevatorApi.tcp_lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C38A7450>>
[INFO]2021-03-03 15:09:37,833: create lock <lock name:elevatorApi.g_lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C389FD50>>
[INFO]2021-03-03 15:09:37,834: create lock <lock name:elevatorMgr.lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C389F1B0>>
[INFO]2021-03-03 15:09:37,853: create lock <lock name:agvserver,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C38EC570>>
[INFO]2021-03-03 15:09:38,137: create lock <lock name:mapMgr.lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C38B2270>>
[INFO]2021-03-03 15:09:38,442: create lock <lock name:agvList.g_agvLock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C3986F30>>
[INFO]2021-03-03 15:09:38,444: create lock <lock name:taskList.lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C3A40900>>
[INFO]2021-03-03 15:09:38,500: charge policy 0 , fix time
[INFO]2021-03-03 15:09:38,531: create lock <lock name:agvApi.g_lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C3A4C510>>
[INFO]2021-03-03 15:09:38,974: start load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-AOI.smap
[DEBUG]2021-03-03 15:09:39,204: elevator thread start
[INFO]2021-03-03 15:09:39,224: load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-AOI.smap ,used 259 ms
[INFO]2021-03-03 15:09:39,231: start load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-DC.smap
[INFO]2021-03-03 15:09:39,293: load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-DC.smap ,used 69 ms
[INFO]2021-03-03 15:09:39,302: start load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-DDT.smap
[INFO]2021-03-03 15:09:39,363: load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-DDT.smap ,used 69 ms
[INFO]2021-03-03 15:09:39,371: start load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-FQC.smap
[DEBUG]2021-03-03 15:09:39,515: create aliveObj SK_DOOR 门控离线异常 timeout 60
[DEBUG]2021-03-03 15:09:39,515: create aliveObj vitualDoor01 门控离线异常 timeout 60
[DEBUG]2021-03-03 15:09:39,515: create aliveObj vitualDoor02 门控离线异常 timeout 60
[INFO]2021-03-03 15:09:39,538: load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-FQC.smap ,used 175 ms
[INFO]2021-03-03 15:09:39,547: start load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-LYH.smap
[INFO]2021-03-03 15:09:39,635: load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-LYH.smap ,used 97 ms
[INFO]2021-03-03 15:09:39,643: start load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-ZP.smap
[INFO]2021-03-03 15:09:39,745: load map D:\python_lx\akm\agvCtrl\projects\akm\AKM-ZP.smap ,used 110 ms
[INFO]2021-03-03 15:09:39,755: start load map D:\python_lx\akm\agvCtrl\projects\akm\office.smap
[INFO]2021-03-03 15:09:39,796: load map D:\python_lx\akm\agvCtrl\projects\akm\office.smap ,used 50 ms
[INFO]2021-03-03 15:09:39,798: import agvDevice.main_agvDevice
[INFO]2021-03-03 15:09:39,799: import alarm.main_alarm
[INFO]2021-03-03 15:09:39,799: import auth.main_auth
[INFO]2021-03-03 15:09:39,807: import button.main_button
[INFO]2021-03-03 15:09:39,810: create lock <lock name:agvApi.g_lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C3FA1C30>>
[INFO]2021-03-03 15:09:39,839: import door.main_door
[INFO]2021-03-03 15:09:39,841: import elevator.main_elevator
[INFO]2021-03-03 15:09:39,842: import virtualDoor.main_doorCtrl
[INFO]2021-03-03 15:09:40,991: create lock <lock name:IOAlarm.s_lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C40F4930>>
[DEBUG]2021-03-03 15:09:40,993: create aliveObj magneticAGV01 二维码掉线异常 timeout 120
[SUCC]2021-03-03 15:09:41,007: utility start, pid=6268, ppid=14296
[DEBUG]2021-03-03 15:09:41,010: start timer
[INFO]2021-03-03 15:09:41,019: load agv file: D:\python_lx\akm\agvCtrl/projects/akm/agv.cfg
[INFO]2021-03-03 15:09:41,019: create lock <lock name:agvList.AGV23.lock,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C45D0F60>>
[DEBUG]2021-03-03 15:09:41,021: create aliveObj AGV23 AGV离线异常 timeout 300
[INFO]2021-03-03 15:09:41,021: AGV23 start agvList run thread
[INFO]2021-03-03 15:09:41,021: create lock <lock name:agv.AGV23.lock_status,<unlocked _thread.RLock object owner=0 count=0 at 0x00000258C45E9600>>
[DEBUG]2021-03-03 15:09:41,022: try connect AGV23 192.168.20.198:19204
[INFO]2021-03-03 15:09:41,022: AGV23 start agvList run2 thread
[WARN]2021-03-03 15:09:41,024: Local server start! version 3.8.0, path D:\python_lx\akm\common, pid 6268
[INFO]2021-03-03 15:09:41,024: add_ignore: taskStatus
[INFO]2021-03-03 15:09:45,069: magneticAGV01 (0, 0, 0, 0)
[INFO]2021-03-03 15:09:48,085: magneticAGV01 (0, 0, 0, 0)
[ERROR]2021-03-03 15:09:51,022: connect AGV23 failed timed out
[DEBUG]2021-03-03 15:09:51,023: try connect AGV23 192.168.20.198:19204
[INFO]2021-03-03 15:09:51,092: startCharge is false
[INFO]2021-03-03 15:09:51,096: magneticAGV01 (0, 0, 0, 0)
[INFO]2021-03-03 15:09:54,104: magneticAGV01 (0, 0, 0, 0)
[DEBUG]2021-03-03 15:09:55,097: GET: http://127.0.0.1:8999/api/switch/off?id=vitualDoor01 addr: 127.0.0.1
[CRITICAL]2021-03-03 15:09:55,106: invoke <function urlRelayclose at 0x00000258C3FEAC10> cause a error: 'dict' object is not callable
pid=6268, ppid=14296, tid=18412
Traceback (most recent call last):
  File "D:\python_lx\akm\common\scadaUtility.py", line 45, in __call
    ret = func(*p,**pp)
  File "D:\python_lx\akm\door\main_door.py", line 24, in urlRelayclose
    switchInfo.close()
  File "D:\python_lx\akm\door\switchMgr.py", line 148, in close
    vitualDoor.vitualDoorClose(self.ip)
  File "D:\python_lx\akm\door\vitualDoor.py", line 53, in vitualDoorClose
    g_swichList(ip).close()
TypeError: 'dict' object is not callable

[DEBUG]2021-03-03 15:09:55,106: used 9ms 200 OK application/json;charset=utf-8, GET: http://127.0.0.1:8999/api/switch/off?id=vitualDoor01
[INFO]2021-03-03 15:09:57,115: magneticAGV01 (0, 0, 0, 0)
[INFO]2021-03-03 15:10:00,122: magneticAGV01 (0, 0, 0, 0)
[ERROR]2021-03-03 15:10:01,025: connect AGV23 failed timed out
[INFO]2021-03-03 15:10:01,025: AGV23 checkLink failed 192.168.20.198 19204
[WARN]2021-03-03 15:10:01,025: agvThread.AGV23[tid:17892] <lock name:agv.AGV23.lock_status,<locked _thread.RLock object owner=17892 count=1 at 0x00000258C45E9600>> ,hold 20003 ms
[INFO]2021-03-03 15:10:01,026: name=agvThread.AGV23, pid=6268, tid=17892
[01] File "C:/Python38/lib/threading.py", line 870, in run() 
[02] File "D:/python_lx/akm/agvCtrl/agvList.py", line 565, in run() 
[03] File "D:/python_lx/akm/driver/seerAgv/agvCtrl.py", line 36, in checkLink() 
[04] File "D:/python_lx/akm/driver/seerAgv/agvApi.py", line 184, in checkLink() 
[05] File "D:/python_lx/akm/common/lock.py", line 102, in release() 
[06] File "D:/python_lx/akm/common/enhance.py", line 268, in getTrace() 
[07] File "D:/python_lx/akm/common/enhance.py", line 248, in getThreadStack()
